### PR TITLE
add small scale to the grown ups (steering) WG

### PR DESCRIPTION
**Pull request by hrhotz**

### CODE REVIEW

File: content/community/steering/index.md [hunk 1]
```diff
@@ -50,12 +50,13 @@ Please contact the committee member leading the project if you would like to lea
 
 * Membership - Dan Blankenberg
 * Misconceptions - Hans-Rudolf Hotz
-* Propeller onboarding cookbook - Anne Fouilloux
 * Global Science - Andrew Lonie
```
Comment (beatrizserrano) [inline content/community/steering/index.md:53]: @fubar2 do we move Global Science to the grown-up section too?

Comment (fubar2) [reply content/community/steering/index.md:53]: It never really get going so perhaps not grown up? 
AFAIK Andrew Lonie won't mind.

Comment (fubar2) [reply content/community/steering/index.md:53]: Links to each grown up project new home ?

Comment (beatrizserrano) [reply content/community/steering/index.md:53]: > Yes please. It never really get going so perhaps not grown up? AFAIK Andrew Lonie won't mind.

ok, then I'll remove it, thanks @fubar2

Comment (beatrizserrano) [reply content/community/steering/index.md:53]: > Links to each grown up project new home ?

they should be there already, is there anything missing?

Comment (fubar2) [reply content/community/steering/index.md]: sorry, didn't look - distracted - now time to eat. Will look soonish.

Comment (fubar2) [reply content/community/steering/index.md]: Looks good to me - thanks @beatrizserrano !

Comment (beatrizserrano) [reply content/community/steering/index.md]: Thank you to you both too!

**Pull request merged**